L1Trigger/Phase2L1ParticleFlow/plugins/L1MHtPFProducer.cc

```diff
@@ -84,8 +84,8 @@ std::vector<l1t::EtSum> L1MhtPfProducer::convertHWToEDM(l1ct::Sum hwSums) const
   mhtVector.SetPhi(l1ct::Scales::floatPhi(hwSums.hwPhi));
   mhtVector.SetEta(0);
 
-  l1t::EtSum ht(htVector, l1t::EtSum::EtSumType::kTotalHt);
-  l1t::EtSum mht(mhtVector, l1t::EtSum::EtSumType::kMissingHt);
+  l1t::EtSum ht(htVector, l1t::EtSum::EtSumType::kTotalHt, hwSums.hwSumPt.V);
+  l1t::EtSum mht(mhtVector, l1t::EtSum::EtSumType::kMissingHt, hwSums.hwPt.V, 0, hwSums.hwPhi.V);
```

Comment:
I think `.to_int()` would be better than `.V` (for sure more readable)

Reply:
This is the only comment I implemented differently. `.to_int()` and `.V` do different things. e.g. for some `ap_fixed<4,2> x = 1.5;`, `.to_int()` basically casts to `int` yielding `1`, while `.V` gives the underlying bits of `6` in this case. In any case, I found a method `bits_to_uint64()` that gives access to the underlying bits, and is more readable. For these `pt_t` variables it would be like doing `hwPt * 4`.